openmdao/docs/feature_reference/core_features/defining_components/metadata.rst

```diff
@@ -0,0 +1,82 @@
+.. _component_metadata:
+
+********************************************
+Component metadata (arguments to components)
+********************************************
+
+The primary job of a component, whether explicit or implicit, is to define inputs and outputs,
+as well as the mapping that computes the outputs given the inputs.
+Often, however, there are incidental parameters that affect the behavior of the component
+but are not considered input variables in the sense of being computed as an output of another component.
+
+For convenience, OpenMDAO provides a way of declaring these parameters, which are referred to as *metadata*.
```

Comment:
I think some clarification is needed here, since we're telling them to do something very non-standard to declare init-args. 

Perhaps its just the "for convenience" that I don't like.

Reply:
Elaborated on how metadata can be set by passing in during instantiation